### PR TITLE
add classic form-layout docs

### PR DESCRIPTION
**Pull request by sosa-vaadin**
Add mention of the classic component version of FormLayout.

### CODE REVIEW

File: articles/flow/upgrading/classic-components.asciidoc [hunk 1]
```diff
@@ -95,6 +95,19 @@ Related to the sizes, `VerticalLayout` and `HorizontalLayout` use Lumo variables
 
 |===
 
+
+=== FormLayout
+
+The classic `FormLayout` resembles, as close as possible, the Vaadin 7 or 8 version of `FormLayout`.
```
Comment (dougjewsbury) [inline articles/flow/upgrading/classic-components.asciidoc:101]:
```suggestion
The classic `FormLayout` resembles, as closely as possible, the Vaadin 7 or 8 version of `FormLayout`.
```

**Review by dougjewsbury — COMMENTED** [review]
Looks OK in terms of AsciiDoc.

**Comment by jouni** [comment]
Note, that the "Flow / Upgrading" section has been removed since this branch was started. Upgrading instructions are now on the top level.  That said, I wonder if the Classic Components should be documented there, or some other place under Flow? @tarekoraby, do you have any thoughts about that?

**Review by Haprog — APPROVED** [review]

File: articles/flow/upgrading/classic-components.asciidoc (hunk 1 above)
Comment (jouni) [reply articles/flow/upgrading/classic-components.asciidoc:101]: @sosa-vaadin, please implement this suggestion.

**Pull request merged**